config/locales/en/admin.yml

```diff
@@ -72,7 +72,6 @@ en:
           allow_welsh_translation: Allow Welsh translation
           enable_ccms_submission: Allow CCMS submissions
           enable_evidence_upload: Enable the new evidence upload feature
-          partner_means_assessment: Enable Partner Means Assessment
```

Comment:
Reminder to remove the relevant hint text, now on line 87

Reply:
👍 thanks!